packages/blocks/src/lib/maplibre/marker.svelte

```diff
@@ -0,0 +1,41 @@
+<!--
+  @component
+
+  Creates a marker on a maplibre map. Must be a child of `<MapLibre>`.
+  
+  ```svelte
+    <MapLibreMarker
+      lngLat={{ lng: 0, lat: 0 }}
+      scale={1}
+      visible={true}
+      color={'blue'}
+    />
+  ```
+-->
+<script lang='ts'>
+
+import { onDestroy } from 'svelte';
+import { type LngLatLike, Marker } from 'maplibre-gl';
+import { useMapLibre } from './hooks';
+
+export let lngLat: LngLatLike;
+export let scale = 1;
+export let visible = true;
+export let color = '#ff0047';
+
+const { map } = useMapLibre()
+
+const marker = new Marker({ scale, color });
```

Comment:
What happens if the `scale` or `color` props change?

Reply:
Didn't make them reactive - will do 👍